### PR TITLE
First version of new JSF tests (create JSF Project, Deploy JSF Project)

### PR DESCRIPTION
**Pull request by rhopp**

### CODE REVIEW

**Comment by psrna** [comment]
testPR

**Comment by rawagner** [comment]
testPR

**Comment by psrna** [comment]
Please add copyright info where missing. thanks!

**Comment by rhopp** [comment]
test this

**Comment by rhopp** [comment]
test this

**Comment by rhopp** [comment]
Ready to review, @psrna .

File: plugins/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/JSFNewProjectSecondPage.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package org.jboss.tools.jsf.reddeer.ui;
```
Comment (psrna) [inline plugins/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/JSFNewProjectSecondPage.java]: Can you please add copyright to this file, too? thx!

Comment (rhopp) [inline plugins/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/JSFNewProjectSecondPage.java]: Sure. np

Comment (rhopp) [inline plugins/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/JSFNewProjectSecondPage.java]: done ;-)

**Pull request merged**